src/components/accordion/_macro.spec.js

```diff
@@ -4,64 +4,65 @@ import * as cheerio from 'cheerio';
 
 import axe from '../../tests/helpers/axe';
 import { renderComponent } from '../../tests/helpers/rendering';
-
-const EXAMPLE_ACCORDION_WITH_TWO_ITEMS = {
-    id: 'accordion-identifier',
-    itemsList: [
-        {
-            title: 'Title for item 1',
-            content: 'Content for item 1',
-        },
-        {
-            title: 'Title for item 2',
-            content: 'Content for item 2',
-        },
-    ],
-};
-
-const EXAMPLE_ACCORDION = {
-    ...EXAMPLE_ACCORDION_WITH_TWO_ITEMS,
-    allButton: {
-        open: 'Open label',
-        close: 'Close label',
-    },
-};
-
```

Comment:
The only thing I am noticing is that we are not longer testing the accordion component with the allButton parameter(so the one buttone that opens all accordions) against axe. Was this intentional?

Reply:
In main `EXAMPLE_ACCORDION` is only used for the jest-axe test. The rest of the time the `allButton` param is added again without using `EXAMPLE_ACCORDION`. This PR still tests those but also adds them so its all covered and fixes the near redundant example set out here with the `allButton` param. I think the only think that is missing is the fact we now don't use the `allButton` version for the jest-axe test. We should add this to the test.